src/gc/os.d

```diff
@@ -69,13 +111,22 @@ else static if (is(typeof(malloc)))
     version = GC_Use_Alloc_Malloc;
 else static assert(false, "No supported allocation methods available.");
 +/
+/**
+ * Indicates if an implementation support fork().
+ *
+ * The value shown here is just demostrative, the real value is defined based
+ * on the OS it's being compiled in.
+ * enum HAVE_FORK = true;
+*/
 
 static if (is(typeof(VirtualAlloc))) // version (GC_Use_Alloc_Win32)
 {
+    enum { HAVE_FORK = false }
```

Comment:
* Use lower case letters and camel casing
* You can write the above with the following syntax: `enum haveFork = false`.